IoTCoreDefaultApp/IoTCoreDefaultApp/Views/Settings.xaml.cs

```diff
@@ -254,5 +388,548 @@ private void RefreshButton_Click(object sender, RoutedEventArgs e)
+            {
+                formatString = BluetoothDeviceInformationDisplay.GetResourceString("BluetoothUnpairingFailureFormat");
+                confirmationMessage = string.Format(formatString, unpairingResult.Status.ToString(), deviceInfoDisp.Name, deviceInfoDisp.Id);
+            }
+            // Display the result of the pairing attempt
+            DisplayMessagePanel(confirmationMessage, MessageType.InformationalMessage);
+
+            // Clear any devices in the listand stop and restart the watcher to ensure state is reflected in list
+            bluetoothDeviceObservableCollection.Clear();
+            StopWatcher();
+            StartWatcher();
+
+            // Re-enable the unpair button
+            unpairButton.IsEnabled = true;
+        }
+
+        /// <summary>
+        /// User has entered a PIN and pressed <Return> in the PIN entry flyout
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void PinEntryTextBox_KeyDown(object sender, Windows.UI.Xaml.Input.KeyRoutedEventArgs e)
+        {
+
+            if (e.Key == Windows.System.VirtualKey.Enter)
+            {
+                //  Close the flyout and save the PIN the user entered
+                TextBox bluetoothPINTextBox = sender as TextBox;
+                string pairingPIN = bluetoothPINTextBox.Text;
+                if (pairingPIN != "")
+                {
+                    // Hide the flyout
+                    inProgressPairButton.Flyout.Hide();
+                    inProgressPairButton.Flyout = null;
+                    // Use the PIN to accept the pairing
+                    AcceptPairingWithPIN(pairingPIN);
+                }
+            }
+        }
+
+        private void AcceptPairingWithPIN(string PIN)
+        {
+            if (pairingRequestedHandlerArgs != null)
+            {
+                pairingRequestedHandlerArgs.Accept(PIN);
+                pairingRequestedHandlerArgs = null;
+            }
+            // Complete the deferral here
+            CompleteDeferral();
+        }
+
+        /// <summary>
+        /// Call when selection changes on the list of discovered Bluetooth devices
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ResultsListView_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+
+        }
+
+        /// <summary>
+        /// Get the set of acceptable ceremonies from the check boxes
+        /// </summary>
+        /// <returns></returns>
+        private DevicePairingKinds GetSelectedCeremonies()
+        {
+            DevicePairingKinds ceremonySelection = DevicePairingKinds.ConfirmOnly | DevicePairingKinds.DisplayPin | DevicePairingKinds.ProvidePin | DevicePairingKinds.ConfirmPinMatch;
+            //if (confirmOnlyOption.IsChecked.HasValue && (bool)confirmOnlyOption.IsChecked) ceremonySelection |= DevicePairingKinds.ConfirmOnly;
```

Comment:
either add comments to describe why all of this unused code is in the file or get rid of it. (I would prefer getting rid of it)